specs/online_order_spec.rb

```diff
@@ -3,88 +3,114 @@
   describe "#initialize" do
     it "Is a kind of Order" do
-      # Check that an OnlineOrder is in fact a kind of Order
+      all_orders = Grocery::OnlineOrder.all
 
-      # Instatiate your OnlineOrder here
-      # online_order =
-      # online_order.must_be_kind_of Grocery::Order
-    end
+      all_orders.class.must_equal Array
 
-    it "Can access Customer object" do
-      # TODO: Your test code here!
-    end
+      all_orders.each do |order|
+        order.must_be_instance_of Grocery::OnlineOrder
+      end
 
-    it "Can access the online order status" do
-      # TODO: Your test code here!
+      all_orders.count.must_equal 100
     end
   end
 
-  describe "#total" do
-    it "Adds a shipping fee" do
-      # TODO: Your test code here!
-    end
+  xit "Can access Customer object" do
+    # TODO: I did not create a Customer class.
+  end
+
+  it "Can access the online order status" do
+
+    all_orders = Grocery::OnlineOrder.all
+
+    all_orders[0][3].must_be_kind_of Symbol
 
-    it "Doesn't add a shipping fee if there are no products" do
-      # TODO: Your test code here!
-    end
   end
+end
 
-  describe "#add_product" do
-    it "Does not permit action for processing, shipped or completed statuses" do
-      # TODO: Your test code here!
-    end
+xdescribe "#total" do
+  xit "Adds a shipping fee" do
+    # TODO: Your test code here!
+  end
 
-    it "Permits action for pending and paid satuses" do
-      # TODO: Your test code here!
-    end
+  xit "Doesn't add a shipping fee if there are no products" do
+    # TODO: Your test code here!
   end
+end
 
-  describe "OnlineOrder.all" do
-    it "Returns an array of all online orders" do
-      # TODO: Your test code here!
-    end
+xdescribe "#add_product" do
+  xit "Does not permit action for processing, shipped or completed statuses" do
+    # TODO: Your test code here!
+  end
 
-    it "Returns accurate information about the first online order" do
-      # TODO: Your test code here!
-    end
+  xit "Permits action for pending and paid satuses" do
+    # TODO: Your test code here!
+  end
+end
 
-    it "Returns accurate information about the last online order" do
-      # TODO: Your test code here!
+xdescribe "OnlineOrder.all" do
+  it "Returns an array of all online orders" do
+    all_orders = Grocery::OnlineOrder.all
+    # all_orders.must_be_kind_of Array
+    all_orders.class.must_equal Array
+
+    all_orders.each do |order|
+      order.must_be_instance_of Grocery::OnlineOrder
     end
+
+    all_orders.count.must_equal 100
   end
 
-  describe "OnlineOrder.find" do
-    it "Will find an online order from the CSV" do
-      # TODO: Your test code here!
-    end
 
-    it "Raises an error for an online order that doesn't exist" do
-      # TODO: Your test code here!
-    end
+it "Returns accurate information about the first online order" do
+  all_orders = Grocery::OnlineOrder.all
+
+  all_orders[0].id.must_equal 1
+
+  all_orders[0].products.must_be_kind_of Hash
```

Comment:
You could also check if the products contains "Annatto seed" at 58.38 or something similar